docs/ref/forms/api.txt

```diff
@@ -1210,24 +1210,33 @@ Methods of ``BoundField``
 
 .. method:: BoundField.label_tag(contents=None, attrs=None, label_suffix=None)
 
+    Renders the label tag of a form field with the template specified by
```

Comment:
`with the` -> `using the`?
I'm also torn on whether `the label tag of a form field` should be `a label tag for the form field` ... either is probably fine tbh.

Reply:
Ok, I'll go with `a label`.